pages/pipelines/functions.rst

```diff
@@ -508,6 +525,11 @@ processed for this message.
 
 If ``remove_from_default`` is ``true``, the message is also removed from the default stream "All messages".
 
+Example::
+
+        // Route the current processed message to a stream named `Custom Stream`
+        route_to_stream("Custom Stream");
```

Comment:
using the name is onyl the second best option - could you also add example for the route to a streamid.

From a performance perspective this is prefered and in addition the name can be assigned multiple times. The ID is uniq.

Reply:
Done. I've never actually used the ID. I believe that it goes in quotes, is that correct? Add it that way but feel free to fix if incorrect.